.idea/misc.xml

```diff
@@ -3,6 +3,9 @@
   <component name="DesignSurface">
     <option name="filePathToZoomLevelMap">
       <map>
+        <entry key="..\:/Users/thiba/Documents/GitHub/House-Organizer/app/src/main/res/drawable/add_person.xml" value="0.2245" />
+        <entry key="..\:/Users/thiba/Documents/GitHub/House-Organizer/app/src/main/res/layout/activity_edit_household.xml" value="1.0" />
+        <entry key="..\:/Users/thiba/Documents/GitHub/House-Organizer/app/src/main/res/layout/calendar_upcoming_row.xml" value="0.17831813576494426" />
```

Comment:
Those lines are better not here, although I know it is hard to avoid them when committing xD

Reply:
oh yea my bad i keep forgetting about them, ill delete them in the next commits !